src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -123,45 +121,54 @@ public void Mutate()
 
         private void CompileMutations()
         {
-            using (var ms = new MemoryStream())
+            using var ms = new MemoryStream();
+            using var msForSymbols = _options.DevMode ? new MemoryStream() : null;
+            // compile the mutated syntax trees
+            var compileResult = _compilingProcess.Compile(_input.ProjectInfo.ProjectContents.CompilationSyntaxTrees, ms, msForSymbols, _options.DevMode);
+
+            foreach (var testProject in _input.ProjectInfo.TestProjectAnalyzerResults)
             {
-                // compile the mutated syntax trees
-                var compileResult = _compilingProcess.Compile(_input.ProjectInfo.ProjectContents.CompilationSyntaxTrees, ms, _options.DevMode);
+                var injectionPath = testProject.TargetDirectory;
+                if (!_fileSystem.Directory.Exists(injectionPath))
+                {
+                    _fileSystem.Directory.CreateDirectory(injectionPath);
+                }
 
-                foreach (var testProject in _input.ProjectInfo.TestProjectAnalyzerResults)
+                // inject the mutated Assembly into the test project
+                using (var fs = _fileSystem.File.Create(Path.Combine(injectionPath, _input.ProjectInfo.ProjectUnderTestAnalyzerResult.TargetFileName)))
```

Comment:
Since you are using the new using declaration above, it would be nice to also use it here

Reply:
Done.